src/main/java/mapfood/model/MotoboyWithDistance.java

```diff
@@ -0,0 +1,14 @@
+package mapfood.model;
```

Comment:
Vamos pensar em um modo melhor de colocarmos a distância, precisamos pensar um pouquinho onde vai fazer mais sentido a distância ficar.

Reply:
Você já viu como retorna a query quando faz a agregação?
se n for assim, sinceramente nem sei como vai ser, mas fique a vontade